### DISCUSSION TITLE
Is there a way to add points per team prediction?

### DISCUSSION THREAD

**Discussion by wachna87** [kyleskom/NBA-Machine-Learning-Sports-Betting#429] [Q&A]
Hi everyone. I am curious if there is a way to add a points per team prediction to the script?
You get a prediction if a game will go O/U and a prediction for a winner team, so it would be cool to get a prediction regarding the team points. I would like to check if you can use this for spread betting aswell

**Comment by ear361** [comment]
Yes if you update the code. Both the points and spread data are available.

**Reply by wachna87** [reply]
thx for your answer. Is there any explanation on how to update the code.
I am finally able to run the main.py with wsl in Linux so I can adjust the models, scripts and whatever is needed.
But I dont know where exactly I should make some changes.

**Comment by whiskey1906** [comment]
how do I get access to this code to utilize
